examples/tts/conf/fastpitch_align.yaml

```diff
@@ -0,0 +1,183 @@
+name: "FastPitch"
+sample_rate: 44100
+train_dataset: ???
+validation_datasets: ???
+prior_folder: ???
+
+model:
+  learn_alignment: true
+  n_speakers: 1
+  symbols_embedding_dim: 384
+  max_token_duration: 75
+  n_mel_channels: 80
+  pitch_embedding_kernel_size: 3
+  n_window_size: 2048
+  n_window_stride: 512
```

Comment:
I would suggest to convert all values related to audio params according to sample_rate=22050.

Reply:
done